test/ClientTest.cpp

```diff
@@ -712,12 +712,9 @@ test_sigpipe(Connector<BUFFER, NetProvider> &client)
 	 * Create a large payload so that request needs at least 2 `send`s, the
 	 * latter being written to a closed socket.
 	 */
-	rid_t f = conn.space[0].replace(std::vector<uint64_t>(100000, 777));
+	rid_t f = conn.space[0].replace(std::vector<uint64_t>(1000000, 777));
 	fail_if(client.wait(conn, f, WAIT_TIMEOUT) == 0);
-#ifdef __FreeBSD___
-	/* FIXME(gh-53) reenable the test if possible. */
 	fail_unless(conn.getError().saved_errno == EPIPE);
-#endif
```

Comment:
Let's wait for `EPIPE` on Linux and for any appropriate error (`ECONNRESET`) on other platforms.

Reply:
Fixed as we discussed.

Reply:
macOS 12 has failed - send returned an error `Resource temporarily unavailable`. AFAIU, is it `EAGAIN`.